### ISSUE TITLE
Communicator::sum(T&) not defined

### ISSUE THREAD

**Issue by BalticPinguin** [libMesh/libmesh#2240]
Hi,
I am currently trying to run the newest version; but the checks failed for me:
```
from ../../../../master/examples/subdomains/subdomains_ex3/subdomains_ex3.C:32:
../../../include/libmesh/communicator.h:328:8: warning: inline function ‘void libMesh::Parallel::Communicator::sum(T&) const [with T = double]’ used but never defined
   void sum(T & r) const;
```
I think, this might be due to the new 'inline'-character from commit 7b58040354 (sorry; I didn't find the PR-nr now) ? At least, [this discussion](
https://stackoverflow.com/questions/9338152/must-the-definition-of-a-c-inline-functions-be-in-the-same-file) supports this. However, this might have shown up in some other checks already?

**Comment by jwpeterson** [comment]
Hi, 

You should be able to fix the issue by adding
```
#include "libmesh/parallel.h"
```
to that example? I'm not really sure why our `--disable-strict-lgpl` CI testing missed this... it seems to work fine for the example in question. Let us know if including that header fixes it and we'll apply the fix in upstream.

**Comment by roystgnr** [comment]
Sorry, this was my fault.  It should have been fixed yesterday, though, when we merged #2238 - is it still failing for you after that?

**Comment by BalticPinguin** [comment]
thanks; you were right: now the issue is resolved.
